src/Renderer/Shader/PointsFS.glsl

```diff
@@ -18,9 +18,12 @@ void main() {
 
 // Early discard (clipping planes and shape)
 #include <clipping_planes_pars_fragment>
+    if (vColor.a == 0.0) {
+        discard;
+    }
     if (shape == PNTS_SHAPE_CIRCLE) {
         //circular rendering in glsl
-        if ((length(gl_PointCoord - 0.5) > 0.5) || (vColor.a == 0.0)) {
```

Comment:
this vColor.a == 0.0
Has no reason to be here indeed (as there is no reason to be apply only on shape == Circle)
But should it be check on all point to be discarded or  not ?

As  @Desplandis said, it's the property transparent of the material whould should be set to true when one class or more has an opacity that is not 1.

Reply:
I agree. I removed the early discard and applied suggested fix.